### PR TITLE
presenting the Oscar winning welcome page

### PR DESCRIPTION
**Pull request by cezaraugusto**
closes https://github.com/brave/brave-browser/issues/1847

`npm run test-unit` should cover edge cases of welcome page
`brave://welcome` for manual testing

oscar

all props to @rossmoody I just did the code

### CODE REVIEW

**Comment by bbondy** [comment]
@cezaraugusto ping on this since this is 22 days old

File: components/brave_welcome_ui/BUILD.gn [hunk 1]
```diff
@@ -8,12 +8,12 @@ transpile_web_ui("brave_welcome_ui") {
...
-    "components/footer.tsx",
-    "components/importScreen.tsx",
-    "components/rewardsScreen.tsx",
-    "components/shieldsScreen.tsx",
+    "components/screens/footerBox.tsx",
+    "components/screens/importBox.tsx",
+    "components/screens/rewardsBox.tsx",
+    "components/screens/searchBox.tsx",
+    "components/screens/shieldsBox.tsx",
+    "components/screens/welcomeBox.tsx",
```
Comment (petemill) [inline components/brave_welcome_ui/BUILD.gn]: `themeBox` is missing. Please check that BUILD.gn is up to date. Here's why https://github.com/brave/brave-browser/issues/2363

File: components/img/welcome/welcome_import.svg [hunk 2]
```diff
@@ -0,0 +1,164 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 325 200">
```
[the remark this answers is not in the document]
Comment (rossmoody) [reply components/img/welcome/welcome_import.svg]: can speak to this one cause i did it. these were run through svgo but they are pretty serious svgs. quite a few elements with linear gradients in each.

**Comment by rossmoody** [comment]
@cezaraugusto Per animation notes: The initial hope was to give the user a few seconds to adjust (since technically this would be the first time opening Brave) and also the panels are pretty big but I think I'd prefer to be a little fast than a little slow and we can always fine tune after this. Could you adjust the transition timing for the panels and the initial reveal to 400ms? We can keep the ease designations the same.

**Comment by petemill** [comment]
0.60.x 4bf00c48

**Comment by petemill** [comment]
@rebron @kjozwiak @srirambv uplift request for 59 (or 58)

**Comment by cezaraugusto** [comment]
@petemill approved for 59 per https://bravesoftware.slack.com/archives/C7VLGSR55/p1544639764749500

**Comment by kjozwiak** [comment]
@petemill uplift request to `0.59.x beta` approved 👍 Please add/remove all needed labels and ensure that the associated issue is moved to the correct milestone.

**Comment by cezaraugusto** [comment]
0.59.x cb0fd9b8

**Pull request merged**